docs/usage/screening.md

```diff
@@ -58,7 +58,17 @@ We recommend to run MAGeCK MLE and BAGEL2 as these are the most used and most re
 
 ### Running CRISPRcleanR
 
-CRISPRcleanR is used for gene count normalization and the removal of biases for genomic segments for which copy numbers are amplified. Currently, the pipeline only supports annotation libraries already present in the R package and which can be found [here](https://github.com/francescojm/CRISPRcleanR/blob/master/Reference_Manual.pdf). To use CRISPRcleanR normalization, use `--crisprcleanr library`, `library` being the exact name as the library in the CRISPRcleanR documentation (e.g: "AVANA_Library").
+CRISPRcleanR is used for gene count normalization and the removal of biases for genomic segments for which copy numbers are amplified. Currently, the pipeline supports annotation libraries already present in the R package or a annotation file the user can provide.
+Most used library already have an annotation dataset which you can find [here](https://github.com/francescojm/CRISPRcleanR/blob/master/Reference_Manual.pdf). To use CRISPRcleanR normalization, use `--crisprcleanr library`, `library` being the exact name as the library in the CRISPRcleanR documentation (e.g: "AVANA_Library").
+Otherwise, if you wish to provide your own file, please provide it in csv form, and make sure it follows the following format :
+
+| ,CODE                | GENES             | EXONE | CHRM | STRAND | STARTpos  | ENDpos    |
```

Comment:
is there a column header missing?

Reply:
no it's a bit of a weird format, i also was confused, maybe i should accentuate it

Reply:
If it looks good on the rendered Markdown it's enough for me 👍